articles/extensions/delegated-admin/hooks.md

```diff
@@ -225,15 +240,17 @@ function(ctx, callback) {
     return callback(new Error('You can only create users within your own department.'));
   }
 
+  if (ctx.method === 'update') {
+    /* If updating, only set the fields we need to send */
+    Object.keys(newProfile).forEach(function(key) {
+      if (newProfile[key] === ctx.request.originalUser[key]) delete newProfile[key];
+    });
+  }
+
+  // NOTE: if you are setting userFields with restricted values using the settings query, you should enforce their limits here using the ctx.userFields array
```

Comment:
> NOTE: If you are using the settings query to popular userFields with restricted values, enforce their limits here using the ctx.userFields array

Also, I'm not sure what restricted values are, so maybe a short 3-5 word definition would be helpful?